### PR TITLE
Added onmyway133 and picked some apps

### PR DESCRIPTION
**Pull request by onmyway133**
I would like to help

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,10 +91,9 @@ If you’re participating in *Indie Support Weeks* and want to help promote some
 | [Diagram.Codes Studio](https://studio.diagram.codes) | [@hugozap](https://twitter.com/hugozap) | |
 | [Digital Photo Frame](https://itunes.apple.com/us/app/digital-photo-frame-pro-slideshow-creator/id1219786089?ls=1&mt=8) | [@manuelescrig](https://twitter.com/manuelescrig) | |
 | [Do I need it?](https://apps.apple.com/app/do-i-need-it/id1495111840) | [@maurogarcia_19](https://twitter.com/maurogarcia_19) | |
-| [DrawRun - Easily start running](https://apps.apple.com/us/app/drawrun-map-your-runs/id1442873786) | [@jordibruin](https://twitter.com/jordibruin) | SwiftLee |
+| [DrawRun - Easily start running](https://apps.apple.com/us/app/drawrun-map-your-runs/id1442873786) | [@jordibruin](https://twitter.com/jordibruin) | |
```
Comment (JohnSundell) [inline README.md]: Please revert these removals of others' markings 🙂

Comment (JohnSundell) [reply README.md]: It may just be a Git thing, or a mistake, so no worries, but I'd really appreciate it if you could fix these 🙂

Comment (onmyway133) [reply README.md]: Opps, I think it's because of git  issue. I will make another PR

Comment (JohnSundell) [reply README.md]: Thanks 🙂

**Pull request closed without merging**